x-pack/plugins/ml/public/application/components/anomalies_table/anomaly_details_utils.tsx

```diff
@@ -276,5 +315,342 @@ export function getDetailsItems(anomaly: AnomaliesTableRecord, filter: EntityCel
     });
   }
 
-  return items;
+  return (
+    <>
+      {items.map(({ title, description }) => (
+        <>
+          <EuiFlexGroup gutterSize="none">
+            <EuiFlexItem css={{ width: '180px' }} grow={false}>
+              {title}
+            </EuiFlexItem>
+            <EuiFlexItem>{description}</EuiFlexItem>
+          </EuiFlexGroup>
+          <EuiSpacer size="xs" />
+        </>
+      ))}
+    </>
+  );
+};
+
+export const AnomalyExplanationDetails: FC<{ anomaly: AnomaliesTableRecord }> = ({ anomaly }) => {
+  const explanation = anomaly.source.anomaly_score_explanation;
+  if (explanation === undefined) {
+    return null;
+  }
+  const initialScore = Math.floor(1000 * anomaly.source.initial_record_score) / 1000;
+  const finalScore = Math.floor(1000 * anomaly.source.record_score) / 1000;
+  const scoreDifference = initialScore - finalScore;
+  const ACCEPTABLE_NORMALIZATION = 10;
+
+  const yes = i18n.translate('xpack.ml.anomaliesTable.anomalyExplanationDetails.yes', {
+    defaultMessage: 'Yes',
+  });
+  const no = i18n.translate('xpack.ml.anomaliesTable.anomalyExplanationDetails.no', {
+    defaultMessage: 'No',
+  });
+
+  const explanationDetails = [];
+  const anomalyType = getAnomalyType(explanation);
+  if (anomalyType !== null) {
+    explanationDetails.push({
+      title: (
+        <FormattedMessage
+          id="xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.anomalyType"
+          defaultMessage="Anomaly type"
+        />
+      ),
+      description: <>{anomalyType}</>,
+    });
+  }
+
+  if (scoreDifference > ACCEPTABLE_NORMALIZATION) {
+    explanationDetails.push({
+      title: (
+        <EuiToolTip
+          position="left"
+          content={i18n.translate(
+            'xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.recordScoreTooltip',
+            {
+              defaultMessage: 'Record score normalized blah blah blah',
+            }
+          )}
+        >
+          <span>
+            <FormattedMessage
+              id="xpack.ml.anomaliesTable.anomalyDetails.anomalyExplanationDetails.recordScore"
+              defaultMessage="Record score normalized"
+            />
+            <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
+          </span>
+        </EuiToolTip>
+      ),
+      description: (
+        <EuiFlexGroup gutterSize="xs">
+          <EuiFlexItem grow={false}>
+            <RecordScore score={initialScore} />
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>{` -> `}</EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <RecordScore score={finalScore} />
```

Comment:
Could the `finalScore` be less than 1? If so, I think we should format in the same way as we do in other places - `< 1`.

Reply:
We don't do this when showing the record score on the left hand side, we show the full value. We only show `<1` in the table.
![image](https://user-images.githubusercontent.com/22172091/195838173-6b492101-dfc5-4c15-8dcf-660477702ff2.png)